freebasics/templates/base.html

```diff
@@ -35,56 +28,17 @@
   <link rel="stylesheet" href="{% static 'css/template-base.css' %}">
   {% endcompress %}
 
-  {% block extra_css %}
-  {% endblock %}
+  {% block extra_css %}{% endblock %}
   {% custom_css %}
-  <!-- Google Fonts -->
-  <script src="{% static 'js/webfont.js' %}"></script>
   <!-- Google Tag Manager -->
-       <noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-M9FBNX"
-       height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
-       <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-       new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-       j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-       '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-       })(window,document,'script','dataLayer','GTM-M9FBNX');</script>
-       <!-- End Google Tag Manager -->
-
-       <script type="text/javascript">
-              WebFontConfig = {
-                google: { families: [ 'Exo+2:400,700:latin' ] }
-              };
-              (function() {
-                var wf = document.createElement('script');
-                wf.src = ('https:' == document.location.protocol ? 'https' : 'http') +
-                  '://ajax.googleapis.com/ajax/libs/webfont/1/webfont.js';
-                wf.type = 'text/javascript';
-                wf.async = 'true';
-                var s = document.getElementsByTagName('script')[0];
-                s.parentNode.insertBefore(wf, s);
-              })();
-              /*! loadJS: load a JS file asynchronously. [c]2014 @scottjehl, Filament Group, Inc. (Based on http://goo.gl/REQGQ by Paul Irish). Licensed MIT */
-            function loadJS( src, cb ){
-              "use strict";
-              var ref = window.document.getElementsByTagName( "script" )[ 0 ];
-              var script = window.document.createElement( "script" );
-              script.src = src;
-              script.async = true;
-              ref.parentNode.insertBefore( script, ref );
-              if (cb && typeof(cb) === "function") {
-                script.onload = cb;
-              }
-              return script;
-            }
-            // cut the mustard
-            if('querySelector' in document && 'localStorage' in window && 'addEventListener' in window) {
-              loadJS( "{% static 'js/main.js' %}" );
-            }
-            </script>
-
-            {% compress css file enhanced %}
```

Comment:
I think it's a bit of JS that's meant to load JS asynchronously. It then loads a file called `main.js` which is completely empty.

The `enhanced` CSS file isn't used either.